app/src/main/java/org/fossasia/openevent/fragments/TracksFragment.java

```diff
@@ -55,12 +56,15 @@
 
     private SearchView searchView;
 
+    private TextView noTracksView;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'noTracksView' could be replaced by a local variable.](https://www.codacy.com/app/triplez/open-event-android/file/2406630558/issues/source?bid=3233850&fileBranchId=3233897#l59)